src/lib.rs

```diff
@@ -38,7 +38,7 @@
 //! }
 //!
 //! #[pymodule]
-//! fn my_module(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
+//! fn my_module(_py: Python<'_>, m: Bound<'_, PyModule>) -> PyResult<()> {
```

Comment:
I believe this should now looks like this?
```suggestion
//! fn my_module(m: Bound<'_, PyModule>) -> PyResult<()> {
```